src/actions/CurrencyAction.js

```diff
@@ -19,7 +19,7 @@ export const getExchangeRate = async (base, to, provider, time) => {
       const converted = base === 'ETH' ? to : base;
       const key = `ETH-${converted}`.toUpperCase();
       result = await axios.get(`https://api.coinbase.com/v2/prices/${key}/spot?date=${time.toISOString()}`);
-      return base === 'ETH' ? result.data.data.amount : (1 / result.data.data.amount);
+      return to === 'ETH' ? (1 / result.data.data.amount) : result.data.data.amount;
```

Comment:
we really need to be carful and use bigNumbers but probably its best to not change this code more